tests/e2e/test_plans.py

```diff
@@ -0,0 +1,66 @@
+
+from tests.helpers import assert_raises
+from c8.plan.core import PlansServerError
+from conftest import test_data_billing_plan, test_data_update_plan, get_mm_client_instance, test_update_tenant_billing_plan
+
+
+def test_list_billing_plans():
+    client = get_mm_client_instance()
```

Comment:
Please check how I added fixtures to eliminate repeating this line of code
`@pytest.fixture
def setup_client():
    client = get_client_instance()
    return client`

We can call it plans_setup_client since naming need to be different across all projects.